### PR TITLE
Change mobile renderer to URP performant

### PR DESCRIPTION
**Pull request by JucanAndreiDaniel**
# To test
1. Performance for heavy duty stuff near the old camp should be doubled (from 10-15 to 25-30)

# Testing
* [x] Tested with Pico / Quest

### CODE REVIEW

File: ProjectSettings/QualitySettings.asset [hunk 1]
```diff
@@ -149,7 +153,8 @@ QualitySettings:
     terrainBillboardStart: 50
     terrainFadeLength: 5
     terrainMaxTrees: 50
-    excludedTargetPlatforms: []
+    excludedTargetPlatforms:
```
Comment (JaXt0r) [inline ProjectSettings/QualitySettings.asset:156]: Which setting did you set and excluded Android?

File: ProjectSettings/QualitySettings.asset [hunk 2]
```diff
@@ -118,13 +121,14 @@ QualitySettings:
     globalTextureMipmapLimit: 0
     textureMipmapLimitSettings: []
     anisotropicTextures: 1
-    antiAliasing: 0
+    antiAliasing: 4
```
Comment (JaXt0r) [inline ProjectSettings/QualitySettings.asset:124]: 1. Does it add visible quality?
2. Is it excluded on MVR?

File: ProjectSettings/QualitySettings.asset [hunk 3]
```diff
@@ -29,6 +29,7 @@ QualitySettings:
     billboardsFaceCameraPosition: 0
     useLegacyDetailDistribution: 1
     vSyncCount: 0
+    realtimeGICPUUsage: 25
```
Comment (JaXt0r) [inline ProjectSettings/QualitySettings.asset:32]: What's this?

**Review by JaXt0r — COMMENTED** [review]
Some questions.

File: ProjectSettings/QualitySettings.asset (hunk 1 above)
Comment (JucanAndreiDaniel) [reply ProjectSettings/QualitySettings.asset:156]: I left Android to only use the Low quality preset, so it is ignored for the rest

**Comment by JucanAndreiDaniel** [comment]
most of the changes were done by unity, when saving, setting the default values for some variables like realtimeGICPUUsage and antiAliasing
@JaXt0r

**Review by JaXt0r — APPROVED** [review]

**Pull request merged**